src/dcm/agent/tests/unit/test_ossec.py

```diff
@@ -47,24 +48,29 @@
 Nov  3 19:21:18 vagrant sudo: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
 """
 
-EXPECTED_CALLS =[
- call(-1, '', -1, -1, ''),
- call('1446578476.4335', "'SSHD authentication success.'", '3', '5715', ' | Src IP: 10.0.2.2 | User: vagrant | Nov  3 19:21:16 vagrant sshd[19258]: Accepted publickey for vagrant from 10.0.2.2 port 54086 ssh2: RSA dd:3b:b8:2e:85:04:06:e9:ab:ff:a8:0a:c0:04:6e:d6 | '),
- call('1446578476.4685', "'Login session opened.'", '3', '5501', ' | Nov  3 19:21:16 vagrant sshd[19258]: pam_unix(sshd:session): session opened for user vagrant by (uid=0) | '),
- call('1446578480.4949', "'Successful sudo to ROOT executed'", '3', '5402', ' | User: vagrant | Nov  3 19:21:18 vagrant sudo:  vagrant : TTY=pts/1 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/bash | '),
- call('1446578480.5214', "'Login session opened.'", '3', '5501', ' | Nov  3 19:21:18 vagrant sudo: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)')
-]
+
+EXPECTED_CALLS = [
+ call(1446578476.4335, 'SSHD authentication success.', 3, 5715, '2015 Nov 03 19:21:16 vagrant->/var/log/auth.log\nSrc IP: 10.0.2.2\nUser: vagrant\nNov  3 19:21:16 vagrant sshd[19258]: Accepted publickey for vagrant from 10.0.2.2 port 54086 ssh2: RSA dd:3b:b8:2e:85:04:06:e9:ab:ff:a8:0a:c0:04:6e:d6\n\n'),
+ call(1446578476.4685, 'Login session opened.', 3, 5501, '2015 Nov 03 19:21:16 vagrant->/var/log/auth.log\nNov  3 19:21:16 vagrant sshd[19258]: pam_unix(sshd:session): session opened for user vagrant by (uid=0)\n\n'),
+ call(1446578480.4949, 'Successful sudo to ROOT executed', 3, 5402, '2015 Nov 03 19:21:20 vagrant->/var/log/auth.log\nUser: vagrant\nNov  3 19:21:18 vagrant sudo:  vagrant : TTY=pts/1 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/bash\n\n')]
 
 class TestOssec(unittest.TestCase):
 
     def setUp(self):
         test_utils.connect_to_debugger()
-        self.alert_sender = AlertSender(mock.Mock(), mock.Mock())
+        self.alert_sender = MagicMock()
+        with open("/tmp/alerts.log", "w") as f:
+            f.writelines(TEST_STRING_LONG)
 
     def tearDown(self):
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this method static. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.int.enstratius.com/coding_rules#rule_key=python%3AS2325)